### ISSUE TITLE
Need investigation: test freebsd-13.1 rust stable (macos-12, 13.1) always fail

### ISSUE THREAD

**Issue by Xuanwo** [mozilla/sccache#1529]
https://github.com/mozilla/sccache/actions/runs/3866878308/jobs/6591285828#logs

```shell
  VM_VHD_LINK="https://download.freebsd.org/releases/VM-IMAGES/13.1-RELEASE/amd64/Latest/FreeBSD-13.1-RELEASE-amd64.vhd.xz"
  VM_OS_NAME="freebsd"
  VM_OS_TYPE="FreeBSD_64"
  VM_SSH_PORT=2222
  VM_RELEASE=13.1
  
  VM_LOGIN_TAG="64 (freebsd) (tty"
  
  VM_SHUTDOWN_CMD="/sbin/shutdown -p now"
  
  VM_INSTALL_CMD="pkg  install  -y "
  
  VM_RSYNC_PKG="rsync"
  VM_SSHFS_PKG="fusefs-sshfs && kldload  fusefs"
  
  VM_PRE_INSTALL_PKGS="tree"
  
  
  VM_OPTS="conf/freebsd-13.1.opts.txt"
  
  VM_VBOX="./vbox.sh"
  
  VM_VBOX_LINK="https://raw.githubusercontent.com/vmactions/vbox/v0.0.1/vbox.sh"===================Debug Info in VM=============
  /root
  total 36
  drwxr-x---   3 root  wheel   512B Jan  8 11:40 .
  drwxr-xr-x  20 root  wheel   512B Jan  8 11:40 ..
  -rw-r--r--   2 root  wheel   1.0K May 12  2022 .cshrc
  -rw-------   1 root  wheel   2.0K Jan  8 11:40 .history
  -rw-r--r--   1 root  wheel    80B May 12  2022 .k5login
  -rw-r--r--   1 root  wheel   328B May 12  2022 .login
  -rw-r--r--   2 root  wheel   507B May 12  2022 .profile
  -rw-r--r--   1 root  wheel   865B May 12  2022 .shrc
  drwx------   2 root  wheel   512B Sep 14 14:23 .ssh
  lrwxr-xr-x   1 root  wheel    19B Jan  8 11:39 work -> /Users/runner/work/
  root
  .
  └── work -> /Users/runner/work/
  
  1 directory, 0 files
  ================================================
Error: The process '/bin/bash' failed with exit code 1
```

**Comment by sylvestre** [comment]
Seems to be the issue:

```
2023-01-08T13:05:26.6944670Z Please, consider to write a feedback email to pizzamig at FreeBSD dot org
2023-01-08T13:05:26.6944980Z It gives us the opportunity to learn and improve
2023-01-08T13:05:26.7262680Z cannot open 'potpool': no such pool
2023-01-08T13:05:26.7452010Z exec shell: bash run.sh showDebugInfo
```

**Comment by sylvestre** [comment]
@grembo rings a bell ?

**Comment by sylvestre** [comment]
I guess this is this change:
```

2023-01-08T11:40:29.7102770Z =====
2023-01-08T11:40:29.7104380Z Message from pot-0.15.4:
2023-01-08T11:40:29.7104770Z 
2023-01-08T11:40:29.7105280Z --
2023-01-08T11:40:29.7106060Z Now, before to run 'pot init', it would be wise to edit the configuration file
2023-01-08T11:40:29.7106300Z 
2023-01-08T11:40:29.7106890Z Do not edit pot.default.conf, but pot.conf instead.
2023-01-08T11:40:29.7107400Z Configuration files are stored in /usr/local/etc/pot/
2023-01-08T11:40:29.7107590Z 
2023-01-08T11:40:29.7108220Z To upgrade from prevoius versions of pot (<0.6.0) to 0.6.0+ the following steps are suggested:
2023-01-08T11:40:29.7108980Z * stop all your pots
2023-01-08T11:40:29.7109720Z * re-run `pot init`
2023-01-08T11:40:29.7112670Z * if needed, restart pf
2023-01-08T11:40:29.7113580Z * if needed, use `pot update-config` on your pots
2023-01-08T11:40:29.7114130Z 
2023-01-08T11:40:29.7115290Z The variable `pot_list` in the rc.conf file is deprecated.
2023-01-08T11:40:29.7116260Z Use `pot set-attr` to enable the attribute start-at-boot to your pot
2023-01-08T11:40:29.7116680Z 
2023-01-08T11:40:29.7117310Z Non-privileged users need to be invited to group "pot" in order
```

**Comment by grembo** [comment]
> @grembo rings a bell ?

Yes, I can fix it later today 🤦‍♂️

**Comment by sylvestre** [comment]
thanks :)

**Comment by sylvestre** [comment]
@grembo before I saw your message, I was trying and did https://github.com/mozilla/sccache/pull/1530

WDYT?

**Comment by grembo** [comment]
> @grembo before I saw your message, I was trying and did #1530
> 
> WDYT?

Could technically work as well, but you might have issues if the directory isn't owned by group "pot".

I just created #1531 with a fix before seeing you also did a workaround (I also added a fix to pot that will resolve the underlying issue). Basically when adding a check for /opt/pot, the version command wasn't excluded (duh!). The fix simply ignores the return code of pot version and adds it later as well, so as soon as the upstream fix is released, the version number will show up in the right place again.
